features/upgrade/etcd/upgrade.feature

```diff
@@ -13,26 +12,47 @@ Feature: basic verification for upgrade testing
   @arm64 @amd64
   Scenario: etcd-operator and cluster works well after upgrade - prepare
     Given I switch to cluster admin pseudo user
-    Given I obtain test data file "admin/subscription.yaml"
-    When I run the :create client command with:
-      | f | subscription.yaml |
-    Then the step should succeed
-    When I use the "openshift-operators" project
-    Then status becomes :running of exactly 1 pods labeled:
-      | name=etcd-operator-alm-owned |
-    When I use the "default" project
-    Given I obtain test data file "admin/etcd-cluster.yaml"
-    When I run the :create client command with:
-      | f | etcd-cluster.yaml |
-    Then the step should succeed
-    #bugzilla - id=1979550
-    #Then status becomes :running of exactly 3 pods labeled:
-    #  | etcd_cluster=example |
-    When I run the :get admin command with:
-      | resource | EtcdCluster |
-    Then the step should succeed
-    And the output should match:
-      | example.* |
+    Given I store the masters in the :masters clipboard
+    When I use the "openshift-etcd" project
+    Given I run the steps <%= cb.masters.length %> times:
+    """
+    And status becomes :running of 1 pods labeled:
+      | app=etcd |
+    And status becomes :running of 1 pods labeled:
+      | name=etcd-quorum-guard |
+    """
```

Comment:
@sandeepknd the logic is wrong, if cb.masters.length == 3, and if pod number == 1, this step still pass succ, because get same pod 3 times